Source/utils/stdcompat/string_view.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#ifdef __has_include
+#if defined(__cplusplus) && __cplusplus >= 201703L && __has_include(<string_view>)
```

Comment:
According to https://en.cppreference.com/w/User:D41D8CD98F/feature_testing_macros, the correct value is 201606L

```suggestion
#if defined(__cplusplus) && __cplusplus >= 201606L && __has_include(<string_view>)
```

Reply:
It's 201703L in the stl header.  Maybe a comment would be worthwhile here. We risk breaking a build by using the lower value.

Reply:
![image](https://user-images.githubusercontent.com/216339/116798038-ba032b00-aae3-11eb-9caa-4ff74e7d49bf.png)